charts/ingress-nginx/values.yaml

```diff
@@ -768,23 +768,25 @@ revisionHistoryLimit: 10
 ##
 defaultBackend:
   ##
-  enabled: false
+  enabled: true
 
   name: defaultbackend
   image:
-    registry: registry.k8s.io
-    image: defaultbackend-amd64
+    repository: nginx
+    tag: 1.23.1-alpine
     ## for backwards compatibility consider setting the full image url via the repository value below
     ## use *either* current default registry/image or repository format or installing chart by providing the values.yaml will fail
     ## repository:
-    tag: "1.5"
```

Comment:
Are we changing this because the `defaultbackend-amd64` image is either unmaintained or does not use `TLS >= 1.2`?